src/dotnet/commands/dotnet-add/Program.cs

```diff
@@ -0,0 +1,100 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using Microsoft.DotNet.Cli.CommandLine;
+using Microsoft.DotNet.Cli.Utils;
+using System;
+using System.IO;
+using System.Linq;
+using Microsoft.Build.Construction;
+using Microsoft.DotNet.ProjectJsonMigration;
+using NuGet.Frameworks;
+using Microsoft.DotNet.Tools.Add.ProjectToProjectReference;
+
+namespace Microsoft.DotNet.Tools.Add
+{
+    public class AddCommand
+    {
+        public const string HelpText = @".NET Add Command
+
+Usage: dotnet add [options] [object] <command> [[--] <arg>...]]
+
+Arguments:
+  <object>   The object of the operation. If a project file is not specified, it defaults to the current directory.
+  <command>  Command to be executed on <object>.
+
+Options:
+  -h|--help  Show help information
+
+Args:
+  Any extra arguments passed to the command.
```

Comment:
I think you got confused with `dotnet add` and `dotnet add p2p` since you missed offline conversation. That would have to be rather something like: `dotnet add <command> --help` - I will change it to that